Client/reasn-client/apps/web/app/Nav.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import styles from './Nav.module.css';
+
+
+type Link = {
+    label: string
+    href: string
+};
+
+const Nav = () => {
+return(
+<body>
+  <header>
+    <nav
+      className="
+        flex flex-wrap
+        items-center
+        justify-between
+        w-full
+        h-20
+        text-lg text-white
+        bg-black
+        rounded-b-2xl
+      "
+    >
+    <div>
+        <a className="block text-white font-bold text-3xl mx-60">Reasn.</a>
```

Comment:
```suggestion
        <a className="font-bold text-2xl">Reasn.</a>
```
Why do you set it as a block? It's the only child in div, so it shouldn't matter that much. 'text-white' was specified in **nav**, so it will inherit it. Imo 'text-3xl' is too big.

Reply:
It wasn't deleted by my oversight so already deleted and changed text size.